jira-feature-collector/src/main/java/com/capitalone/dashboard/client/project/ProjectDataClientSetupImpl.java

```diff
@@ -182,7 +182,7 @@ public String getTodayDateISO() {
 	/**
 	 * Mutator method for setting today's date in ISO format
 	 */
-	public void setTodayDateISO(String todayDateISO) {
+	public final void setTodayDateISO(String todayDateISO) {
```

Comment:
Pmd warning because it is called from a constructor.

Sent from my iPhone

On Mar 1, 2016, at 07:24, jdamick <notifications@github.com<mailto:notifications@github.com>> wrote:


In jira-feature-collector/src/main/java/com/capitalone/dashboard/client/project/ProjectDataClientSetupImpl.java<https://github.com/capitalone/Hygieia/pull/341#discussion_r54580392>:

> @@ -182,7 +182,7 @@ public String getTodayDateISO() {
>       /**
>        * Mutator method for setting today's date in ISO format
>        */
> -     public void setTodayDateISO(String todayDateISO) {
> +     public final void setTodayDateISO(String todayDateISO) {


why did u change this to final?

-
Reply to this email directly or view it on GitHub<https://github.com/capitalone/Hygieia/pull/341/files#r54580392>.